### PR TITLE
CLOUD-48453 Cluster Details pane stuck after termination

### PR DESCRIPTION
**Pull request by mhmxs**
@doktoric plz review, i disable cluster details open during DELETE_IN_PROGRESS

### CODE REVIEW

File: app/static/js/console.UI.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ function addClusterListPanelJQEventListeners() {
     // Bootstrap carousel as clusters / cluster details / create cluster slider init
     $jq('.carousel').carousel('pause');
     // show cluster details
-    $jq(document).on("click", ".cluster h4 .btn-cluster", function() {
+    $jq(document).on("click", ".cluster h4 #btn-cluster", function() {
```
Comment (doktoric) [inline app/static/js/console.UI.js]: why do you need this?

Comment (mhmxs) [inline app/static/js/console.UI.js]: Because in h4 there are two items with class btn-cluster, and only one needs the click event.

Comment (doktoric) [inline app/static/js/console.UI.js]: but as I see you disable the button when the status is delete in progress

Comment (mhmxs) [inline app/static/js/console.UI.js]: Disable is a class and doesn't means anything from the click event perspective.

Comment (mhmxs) [inline app/static/js/console.UI.js]: That disable class ocures some visual fix

**Pull request merged**